DATA_FORMAT.md

```diff
@@ -0,0 +1,330 @@
+
+Features must also provide a `min` and `max` range property.
+
+```
+--feature1.json--
+{
+    "data": [
+        <feature value for track id 0>,
+        <feature value for track id 1>,
+        <feature value for track id 2>,
+        ...
+    ],
+    "min": <min value for all features>,
+    "max": <max value for all features>
+}
+```
+
+### 5. Centroids
+
+The centroids file defines the center of each track ID in the dataset. For each index `i`, the coordinates are `(x: data[2i], y: data[2i + 1])`.
+Coordinates are defined in pixels in the frame, where the upper left corner of the frame is (0, 0).
+
+```
+--centroids.json--
+{
+    "data": [
+        <x coordinate for track id 0>,
+        <y coordinate for track id 0>,
+        <x coordinate for track id 1>,
+        <y coordinate for track id 1>,
+        ...
+    ]
+}
+```
+
+### 6. Bounds
+
+The bounds file defines the rectangular boundary occupied by each track ID. For each track ID `i`, the minimum bounding box coordinates (upper left corner) are given by
+`(x: data[4i], y: data[4i + 1])`, and the maximum bounding box coordinates (lower right corner) are given by `(x: data[4i + 2], y: data[4i + 3])`.
+
+Again, coordinates are defined in pixels in the image frame, where the upper left corner is (0, 0).
+
+```
+--bounds.json--
+{
+    "data": [
+        <upper left x for track id 0>,
+        <upper left y for track id 0>,
+        <lower right x for track id 0>,
+        <lower right y for track id 0>,
+        <upper left x for track id 1>,
+        <upper left y for track id 1>,
+        ...
+    ]
+}
+```
+
+### 7. Outliers
+
+The outliers file stores marks whether a given track ID should be marked as an outlier using an array of booleans (`true`/`false`). Indices that are `true` indicate outlier values, and are given a unique color in Nucmorph-Colorizer.
+
+```
+--outliers.json--
+{
+    "data": [
+        <whether track id 0 is an outlier>,
+        <whether track id 1 is an outlier>,
+        <whether track id 2 is an outlier>,
+        ...
+    ]
+}
+```
+
+<details>
+<summary><b>[Show me an example!]</b></summary>
+
+---
+
+For example, if a dataset had the following tracks and outliers, the file might look something like this.
+
+| Track # | Track IDs | Outliers |
+| ------- | --------- | -------- |
+| 1       | 0, 1, 4   | 1        |
+| 2       | 2, 3, 5   | 2, 5     |
+
+```
+--outliers.json--
+{
+    "data": [
+        false, // 0
+        true,  // 1
+        true,  // 2
+        false, // 3
+        false, // 4
+        true   // 5
+    ]
+}
+```
+
+---
+
+</details>
+
+## Collections
+
+Collections are defined by an optional JSON file and group one or more datasets for easy access. By default, they should be named `collection.json`.
+
+Collections are an array of JSON objects, each of which define the `name` (an **alias**) and the `path` of a dataset. This can either be a relative path on a file server, or a complete URL.
```

Comment:
I would say somewhere that the datasets in a collection will be presented in the app for easy switching/selection of dataset within collection.